tools/update-locked-requirements

```diff
@@ -67,3 +79,4 @@ fi
 
 compile_requirements requirements/prod.txt $OUTPUT_PROD_FILE
 compile_requirements requirements/dev.txt $OUTPUT_DEV_FILE
+update_requirements_hash_file
```

Comment:
Maybe put this in `var/` so we don't have to deal with it for `.gitignore`?

Reply:
This file actually needs to be checked in git in order for the implementation to work correctly.